spec/std/http/status_spec.cr

```diff
@@ -0,0 +1,82 @@
+require "spec"
+require "http"
+
+describe HTTP::Status do
+  describe "#from_code" do
```

Comment:
This should be called `.from_code` (class methods are prefixed by `.`, instance methods by `#`).

Could you also add a passing spec for a valid but undefined status code?